utils/swift_build_support/swift_build_support/products/product.py

```diff
@@ -429,7 +434,8 @@ def generate_linux_toolchain_file(self, platform, arch):
 
         return toolchain_file
 
-    def generate_toolchain_file_for_darwin_or_linux(self, host_target):
+    def generate_toolchain_file_for_darwin_or_linux(
+            self, host_target, override_deployment_version=None):
```

Comment:
(optional) `override_deployment_version` may imply this applies to Linux

```suggestion
            self, host_target, override_macos_deployment_version=None):
```